NachoSpigot-Server/pom.xml

```diff
@@ -80,59 +80,60 @@
         <dependency>
             <groupId>net.sf.jopt-simple</groupId>
             <artifactId>jopt-simple</artifactId>
-            <version>3.2</version>
+            <version>5.0.4</version>
             <type>jar</type>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>jline</groupId>
             <artifactId>jline</artifactId>
-            <version>2.12.1</version>
+            <version>2.14.6</version>
             <type>jar</type>
             <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>org.xerial</groupId>
             <artifactId>sqlite-jdbc</artifactId>
-            <version>3.7.2</version>
+            <version>3.36.0.3</version>
             <type>jar</type>
             <scope>runtime</scope>
         </dependency>
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <version>5.1.14</version>
+            <!-- < 8.0.16 versions are affected by https://cwe.mitre.org/data/definitions/843.html -->
+            <version>8.0.28</version>
```

Comment:
A bump could cause very old and badly coded plugins to stop working. so i wouldnt go over 5.1.49 but dunno if that still breaks things